### PR TITLE
Mobile Menu

### PR DESCRIPTION
**Pull request by DarkxDev**

### CODE REVIEW

File: scripts.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+let menuBtn = document.querySelector('#menu')
+let mobileMenu = document.querySelector('#mobile_menu')
+let closeIcon = document.querySelector('#close_icon')
+
+menuBtn.addEventListener('click', () => {
+  mobileMenu.style.display = 'block'
+})
```
Comment (Munsa1) [inline scripts.js:7]: - [ ] When the user clicks on the links in the mobile menu, the mobile menu should disappear as this is  requirement, kindly fix this. The current behaviour is that the user is taken to the correct part of the link but when you scroll up the mobile menu is still there at the top of the page

File: styles.css [hunk 2]
```diff
@@ -15,12 +15,91 @@ body {
...
   display: none;
 }
 
+#mobile_menu {
+  background-color: white;
+  display: none;
+  width: 375px;
+  margin: auto;
+  height: auto;
+}
```
Comment (Munsa1) [inline styles.css:31]: - [ ] Kindly ensure your design follows the given design as this a requirement.

Your design
image

Given design
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @DarkxDev, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jerryowusu — APPROVED** [review]
Hi @DarkxDev , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

200w


## Highlights
All linters are passing :heavy_check_mark: 
Great effort in making the required changes :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**